tests/common/sql_test_case.py

```diff
@@ -119,7 +118,7 @@ def teardown_close_warehouses(cls):
         SqlTestCase.warehouse_fixtures_by_target = {}
 
     def sql_update(self, sql: str) -> int:
-        return sql_update(self.warehouse.connection, sql)
+        return self.warehouse.sql_update(sql)
 
     def sql_updates(self, sqls: List[str]):
```

Comment:
I did not have to update this one - like `sql_update` above - to make the local tests pass.